src/style.css

```diff
@@ -342,47 +342,43 @@
   }
 }
 
-.rdp-weeks_next_enter {
```

Comment:
Does it make sense to flip the words in the class names?
Naming the classes like `weeks_enter_next`?

I don't know if it's just for me how I think logically about it, but for me to reason about what is happening in the animation, I need to thnk first about the "entering" or "exiting" month, than if it's before or after, so I usually read "enter" -> "next" to understand the meaning of the class.

Also, what do you think of changing the naming `prev` and `next` to use the naming from `date-fns` for comparing dates, which is `after` and `before`.
IMO reusing the same concept used for other date libs makes it easier to understand.
So with my suggestion, the class I referenced in this comment would be named `rdp-weeks_enter_after`.

Reply:
I do indeed prefer using “before/after”!

However, I’m not convinced that it should be used as a suffix. To me, “before” and “after” are adjectives that modify “weeks” or “caption.” I’m also not entirely sure what it means in this context. :)

I renamed it using before/after in [commit `1744724`](https://github.com/gpbl/react-day-picker/pull/2694/commits/174472439c951913002d0009982b54abf084843a).

Reply:
All good, thanks for updating it 👍 
Yeah, the other part is more personal taste, I find my way to read the classes to create a mental model of how they are used by the DayPicker.